doc/api/tls.md

```diff
@@ -179,11 +180,33 @@ line switch. For instance, the following makes
 node --tls-cipher-list="ECDHE-RSA-AES128-GCM-SHA256:!RC4"
 ```
 
+The default can also be replaced on a per client or server basis using the
+`ciphers` option from [`tls.createSecureContext()`][], which is also available
+in [`tls.createServer()`], [`tls.connect()`], and when creating new
+[`tls.TLSSocket`]s.
+
+Consult
+<https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT> for
```

Comment:
ok, maybe In a follow on commit, because this style of link to openssl docs is pervasive here and in crypto.md, I can change them all in one go.